docs/cloud-native-security/vuln-management-faq.asciidoc

```diff
@@ -50,3 +50,7 @@ Yes, CNVM scans all EC2 instances, whether they are running or stopped, to ensur
 *What AWS permissions does the user require to run the CloudFormation template for CNVM onboarding?*
 
 To run the CloudFormation template for CNVM onboarding, you need an AWS user account with permissions to perform the following actions: run CloudFormation templates, create IAM Roles and InstanceProfiles, and create EC2 SecurityGroups and Instances.
+
+*Why do I get an error when I try to run the CloudFormation template?*
+
+It's possible you're using an unsupported region. Currently the `eu-north-1` and `af-south-1` are not supported because they don't provide the required instance types.
```

Comment:
```suggestion
It's possible you're using an unsupported region. Currently the `eu-north-1` and `af-south-1` regions are not supported because they don't provide the required instance types.
```